File: app/src/main/java/siergo_o/onlinernews/DescriptionWebView.java
package siergo_o.onlinernews;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.webkit.WebView;

import androidx.appcompat.app.AppCompatActivity;

public class DescriptionWebView extends AppCompatActivity {

    String Url;

    @SuppressLint("SetJavaScriptEnabled")
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.description_web_view);

        WebView webView = findViewById(R.id.webView);
        webView.getSettings().setJavaScriptEnabled(true);

        Bundle extras = getIntent().getExtras();


        if (extras != null) {
            Url = extras.getString("Url");
            // and get whatever type user account id is
        }
        webView.loadUrl(Url);
    }
}

File: app/src/main/java/siergo_o/onlinernews/presentation/base/DescriptionWebView.kt
package siergo_o.onlinernews.presentation.base;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.webkit.WebView;

import androidx.appcompat.app.AppCompatActivity;

import siergo_o.onlinernews.R;

class DescriptionWebView : AppCompatActivity() {

    private var url: String? = ""

    @SuppressLint("SetJavaScriptEnabled")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.description_web_view)
        val webView: WebView = findViewById(R.id.webView)
        webView.settings.javaScriptEnabled = true
        val extras = intent.extras
        if (extras != null) {
            url = extras.getString("Url")
        }
        webView.loadUrl(url)
    }
}
